pro_tes/middleware/middleware.py

```diff
@@ -62,7 +67,15 @@ def modify_request(self, request):
                     )
 
         if self.input_uris:
```

Comment:
No the errors mentioned, do not respond to empty input URI, there was just an outside check, I can remove the outside check and maybe raise a ValueError in `distance.py` with a message ( "No input URI available) and then catch the error hence, removing the if-else block.

Reply:
Yup. It doesn't make sense to use the distance-based logic if there are no inputs, so I think it would be good to raise a `ValueError` and then catch that outside and remove the `if`-`else`.

Reply:
Okay, fixed in the last commit.